src/Transients.h

```diff
@@ -85,6 +85,9 @@ class Transients: public Store::Controlled, public Ipc::StoreMapCleaner
     bool isReader(const StoreEntry &) const;
     /// whether the entry is in "writing to Transients" I/O state
     bool isWriter(const StoreEntry &) const;
+    /// whether there is a writer into the Transients entry, this
```

Comment:
This comment is very confusing. It is easier to read the code than to decipher the Americanization of the compound noun split between beginning and end of the sentence.
Perhapse use the same style as the isWriter() documentation?
```suggestion
    /// whether the StoreEntry is attached anywhere for "writing to Transients" I/O
```

Reply:
Eduard's replacement is OK (after `s/are/is/`). Here are more options to consider if Amos does not like Eduard's replacement:

`whether isWriter() is true for some SMP kid`
`whether there is an SMP kid which isWriter()`
`whether there is an SMP kid for which isWriter() is true`

@yadij, does the latest committed version or any of the above work well enough for you?

Reply:
Latest commit is fine.